source/dotnet/Library/AdaptiveCards.Rendering.Html/AdaptiveRenderContext.cs

```diff
@@ -45,7 +45,6 @@ public HtmlTag Render(AdaptiveTypedElement element)
             }
         }
 
-
         public string GetColor(AdaptiveTextColor color, bool isSubtle)
```

Comment:
Out of curiosity: Question are we not doing xml documentation for our functions/classes etc ?

Reply:
we have xml documentation for some methods and properties, it would be a good idea to start adding it to the majority of the methods

---
In reply to: [272767265](https://github.com/Microsoft/AdaptiveCards/pull/2625#discussion_r272767265) [](ancestors = 272767265)